packages/web-components/src/components/masthead/masthead-menu-button.ts

```diff
@@ -24,12 +28,56 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  * @element dds-masthead-menu-button
  */
 @customElement(`${ddsPrefix}-masthead-menu-button`)
-class DDSMastheadMenuButton extends BXHeaderMenuButton {
+class DDSMastheadMenuButton extends HostListenerMixin(BXHeaderMenuButton) {
+  /**
+   * The name of the custom event fired after the seach is toggled.
+   */
+  static get eventToggle() {
+    return `${ddsPrefix}-masthead-search-toggled`;
+  }
+
   /**
    * The handle for focus wrapping.
    */
   private _hFocusWrap: Handle | null = null;
 
+  /**
+   * Media query listener handler.
+   */
+  private _mqHandler;
```

Comment:
```suggestion
  private _mqHandler: Handle | null = null;
```

Also please add `import Handle from '../../globals/internal/handle';`.